master/reference/advanced/etcd-rbac/kubernetes-advanced.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Advanced Kubernetes set ups utilizing etcd RBAC
+---
+
+When using etcd with RBAC, all components that access etcd must be configured
+with the proper certificates.  This document describes advanced configuration
+options of individual component that can be configured and offers some help
```

Comment:
Maybe this preamble should state upfront that this guide builds on the simple guide to add additional roles to lock down individual Calico components in a Kubernetes deployment.  I don't think anything in this guide alters how we lock down Kubernetes.